src/writeData/containers/ClientLibrariesPage.tsx

```diff
@@ -89,40 +110,43 @@ const ClientLibrariesPage: FC = () => {
     <GetResources
       resources={[ResourceType.Authorizations, ResourceType.Buckets]}
     >
-      <TemplateProvider>
+      <TemplateProvider
+        variables={{token: TOKEN_PLACEHOLDER, bucket: BUCKET_PLACEHOLDER}}
```

Comment:
Here is another example of something that defines these variables. Make it one thing, which is gonna be WriteDataContext